te/tedp_docker/setup.sh

```diff
@@ -70,16 +71,16 @@ cd /tmp/
 rm -rf openssl*
 
 #LIBCURL
-cd curl-7.67.0
```

Comment:
I will check with @vipinpr5991 also one and get back, while using curl-7.67 I was hitting an issue during compilation 
while i also tried curl-7.87 - but the the libcurl api has changed and stopped supporting a few, it would require a lot of C code changes to get this done. The intent was minimum penetration/changes in C code.